base/applications/wordpad/lang/id-ID.rc

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2004 by Krzysztof Foltman
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ * 
+ * Translated by Reinhart Previano K. <reinhart@reinhart1010.id> 2022
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_DEFAULT
```

Comment:
```suggestion
LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT
```
Make sure you have changed `LANG_ENGLISH` to the language you are translating when creating a new language file, or the compiler will complain.

Reply:
Thanks for noticing this!